test/testsets/integrations/RCall_refresh.jl

```diff
@@ -0,0 +1,31 @@
+include("../../utilities/prelude.jl")
+
+@testset "RCall refresh" begin
+    s = Server()
+
+    base_file =
+        joinpath(@__DIR__, "..", "..", "examples", "integrations", "RCall_refresh.qmd")
+    copy = joinpath(dirname(base_file), "RCall_refresh_copy.qmd")
+
+    try
+        # need the file in this dir so the relative --project path is correct
+        cp(base_file, copy)
+        json = run!(s, copy)
+
+        @test json.cells[end].outputs[1].data["text/plain"] == "4.0"
+
+        original = read(copy, String)
+        modified = replace(original, r"delete this block[\s\S]*?```\r?\n" => "")
```

Comment:
Can't this just be
```suggestion
        modified = replace(original, r"x <- 1" => "")
```

? And then get rid of that `delete this block to cause an error in the last cell` in the notebook?

Reply:
I didn't do that because `R""` fails and also cells only containing a comment, that's an upstream issue 
<img width="452" alt="image" src="https://github.com/PumasAI/QuartoNotebookRunner.jl/assets/22495855/95866d5b-3160-40a3-9598-f8a942cd1542">

But I guess I could replace with just `1` or so which doesn't define a variable